src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final String newLine = System.lineSeparator();
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate dateFromFormatter = LocalDate.parse(dateFrom, formatter);
+        LocalDate dateToFormatter = LocalDate.parse(dateTo, formatter).plusDays(1);
```

Comment:
It's not Formatter, Formatter is util to format date, here you have already formatted date
```suggestion
        LocalDate formattedDateFrom = LocalDate.parse(dateFrom, formatter);
        LocalDate formattedDateTo = LocalDate.parse(dateTo, formatter).plusDays(1);
```
or
```suggestion
        LocalDate localDateFrom = LocalDate.parse(dateFrom, formatter);
        LocalDate localDateTo = LocalDate.parse(dateTo, formatter).plusDays(1);
```